packages/conf-ida/conf-ida.0.1/files/find-ida.sh

```diff
@@ -46,7 +46,7 @@ locate_linux() {
 # locate on mac os x doesn't accept -r and -w flags,
 # moreover, we have a better alternative - mfind
 locate_macos() {
-    results=`mdfind -name idaq`
+    results=`mdfind -name idaq | sort -n -r`
```

Comment:
http://www.manpages.info/macosx/sort.1.html says they have the same options for this